com.leinardi.gx52.json

```diff
@@ -0,0 +1,119 @@
+{
+  "app-id": "com.leinardi.gx52",
+  "command": "gx52",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
```

Comment:
x11 only?

Reply:
Oh right, I keep copying this from GWE which is X11 only and I forget about Wayland. Now should be fixed.